File: Python/PGS/LEVEL0/369게임/gunwoo.py
def solution(order):
    answer = 0
    while order > 0:
        if order%10 == 3:
            answer +=1
            
        elif order%10 == 6:
            answer +=1
            
        elif order%10 == 9:
            answer +=1
        order //= 10
    return answer

File: Python/PGS/LEVEL1/정수 제곱근 판별/gunwoo.java
class Solution {
    public long solution(long n) {
        double answer = 0;
        int sq = (int) Math.sqrt(n);
        if(sq == Math.sqrt(n)){
            answer = Math.pow(sq+1,2);
        }
        else{
            answer = -1;
        }
        return (long) answer;
    }
}